homeassistant/components/climate/xs1.py

```diff
@@ -0,0 +1,120 @@
+"""
+Support for XS1 climate devices.
+
+For more details about this platform, please refer to the documentation at
+https://home-assistant.io/components/xs1/
+"""
+import logging
+from functools import partial
+
+from homeassistant.components.climate import (
+    ATTR_TEMPERATURE, ClimateDevice)
+from homeassistant.const import (
+    TEMP_CELSIUS)
+from homeassistant.util.temperature import convert as convert_temperature
+from homeassistant.components.xs1 import ACTUATORS, DOMAIN, SENSORS, \
+    XS1DeviceEntity
```

Comment:
Import should be ordered. Use `isort` if you don't want to do it manually.

Reply:
done